satellite/payments/stripecoinpayments/service.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package stripecoinpayments
+
+import (
+	"context"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go/customer"
+	"gopkg.in/spacemonkeygo/monkit.v2"
+)
+
+var mon = monkit.Package()
+
+// Service is an implementation for PaymentsService via Stripe and Coinpayments
+type Service struct {
+	customers Customers
+}
+
+// NewService is a constructor for PaymentService
+func NewService(customers Customers) *Service {
+	return &Service{
+		customers: customers,
+	}
+}
+
+// Setup creates payment account for selected user
+func (service *Service) Setup(ctx context.Context, userID uuid.UUID, email string) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	params := &stripe.CustomerParams{
+		Email: stripe.String(email),
+	}
+
+	if _, err := customer.New(params); err != nil {
+		return err
```

Comment:
Let's create an error class for wrapping the error returned by Stripe and wrap `err` with it.

Reply:
i would like to do in in separate PR if possible

Reply:
It isn't that much work (i.e. `var ErrorStripe = errs.Class("stripe API error")` and do here `ErrorStripe.Wrap(err)`), but if you want to do a separate PR and you're not going to work right after, please create a ticket and add a TODO here.

For now, at least, return an `errs.Wrap(err)`